Dockerfile.metering-ansible-operator

```diff
@@ -25,7 +25,7 @@ RUN ln -f -s /usr/bin/oc /usr/bin/kubectl
 RUN ln -f -s /tini /usr/bin/tini
 
 RUN pip install --no-cache-dir --upgrade openshift boto3 cryptography
-RUN pip install --upgrade ansible>=2.8
+RUN pip install --upgrade ansible==2.8
```

Comment:
Might be good to add the commit message as a comment here. Later viewers of this segment might not know why we're locking it to this version.

Reply:
Yeah, I was thinking about adding a TODO comment, but I like this better.